version.php

```diff
@@ -26,7 +26,7 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version = 2017101014;
+$plugin->version = 2017101015;
```

Comment:
Let's dont bump up the version just yet. I usually do that right before release.

Reply:
No, it is fine. The version tag has to be updated anyways. It is just that, as the code is going directly to v2.2-stable, there may be epole deploying from there and not from the moodle plugin database. Besides, the other tag has also be updated to 2.2.7.

Keep in mind that this is for the pull request only, once released both will be updated anyways.

And I can revert it directly myself before merging, but it is usually left to the contributor to clean up the code. I contribute with some other open source projects myself, I guess I am just getting used to it :)